build.gradle

```diff
@@ -54,17 +55,17 @@ compileJava {
 }
 
 dependencies {
-    compile('org.seleniumhq.selenium:selenium-java:3.4.0'){
+    compile('org.seleniumhq.selenium:selenium-java:3.5.1'){
```

Comment:
@TikhomirovSergey @mykola-mokhnach i think it only restricts `selenium-java` dependency and not its sub modules or transitive dependency.